doc/Publishing.md

```diff
@@ -87,6 +87,24 @@ For example, if you picked `0.1.0` as a version then you should run:
 
 To confirm that the tagging was correctly done, check the repo's [releases](https://github.com/eclipse-theia/theia/releases) page and confirm the release just done is listed there.
 
+## Updating the release page on eclipse
+- Login to eclipse: https://projects.eclipse.org/projects/ecd.theia
+- On the right side panel, select Release / Create a new release
+    Create a new version ex: 1.2.0
+    Select the edit tab
+    - In "The Basic" section
+        - Select the  "Type A" in the IP Due Diligence type
+    - In the "Project Plan"
+        - Deliverables section
+            - Copy from the changelog.md the section about the updates
+        - Compatibility section
+            - Copy the Braking changes from the changelog.md
```

Comment:
You can directly reference the changelog as a link:

```md
[Changelog](https://github.com/eclipse-theia/theia/blob/master/CHANGELOG.md)
```

Reply:
WE canm put a link to the changelog, but there is only one changelog for all releases. When a user would select the link, it would get the informaion about the current release as well as info for all previous release as well, Is it what we are looking for ?

Reply:
I think it's more helpful and cleaner to link the changelog directly instead of telling whoever reads the docs to go to the changelog.md. The way you phrased it as well does not specify to copy the notes from the specific version anyways.